### PR TITLE
Add Post test

### PR DESCRIPTION
**Pull request by davidporosnicu**

### CODE REVIEW

File: src/tests/components/AddCommentForm.test.js [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+      };
+      wrapper.setState(state);
+      const event = {
+        preventDefault() {}
+      };
+      wrapper.find('#addNewCommentForm').simulate('submit', event);
+      expect(mockAddNewCommentfn).not.toHaveBeenCalled();
+    });
+    //let me know if it's wrong to test both errors in one it
+    it('should set emptyAuthorError and emptyTextError to true if author or text is empty', () => {
```
Comment (horiaradu) [inline src/tests/components/AddCommentForm.test.js]: You could split it into 2 tests.

Comment (davidporosnicu) [reply src/tests/components/AddCommentForm.test.js]: ok

File: src/tests/components/AddCommentForm.test.js [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+        commentDate: new Date(),
+        text: 'new comment',
+        emptyAuthorError: false,
+        emptyTextError: true
+      };
+      wrapper.setState(state);
+      wrapper.instance().changeErrorState();
+      expect(wrapper.state('emptyTextError')).toBe(false);
+    });
+    it('should not set emptyTextError to false if text is empty', () => {
```
Comment (horiaradu) [inline src/tests/components/AddCommentForm.test.js:202]: rename these "not set X to false" into "set X to true"

Comment (davidporosnicu) [reply src/tests/components/AddCommentForm.test.js:202]: changeErrorState = () => {
    const { author, text } = this.state;
    if (author !== '') {
      this.setState({ emptyAuthorError: false });
    }
    if (text !== '') {
      this.setState({ emptyTextError: false });
    }
  };

this function doesn't set anything to true...only to false

Comment (horiaradu) [reply src/tests/components/AddCommentForm.test.js:202]: you are right

**Pull request merged**